src/plots/mapbox/mapbox.js

```diff
@@ -365,6 +365,11 @@ proto.updateLayout = function(fullLayout) {
     } else {
         map.scrollZoom.disable();
     }
+    if(initialView === true) {
```

Comment:
Hmm. Can you move this new block to the `proto.createMap` method instead of making `updateLayout` take two different flavors?

Reply:
I pulled this out into a separate `updateDerived` method in 0ed7584c5f07ca1e9e2c8c8c8eedcb684b686138.